auth/src/contracts/AzureAccount.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import type * as vscode from 'vscode';
+
+export type AzureAccount = Readonly<vscode.AuthenticationSessionAccountInformation>;
```

Comment:
This is just a shortcut type because `vscode.AuthenticationSessionAccountInformation` is long and we use it all over